src/main/java/cloud/fogbow/common/util/SerializableEntity.java

```diff
@@ -0,0 +1,16 @@
+package cloud.fogbow.common.util;
+
+public class SerializableEntity<T> {
+
+    private String className;
+    private String payload;
+
+    public SerializableEntity(T instanceToSerialize) {
+        this.className = instanceToSerialize.getClass().getName();
+        this.payload = GsonHolder.getInstance().toJson(instanceToSerialize);
+    }
+
+    public T getSerializedEntity() throws ClassNotFoundException {
+        return (T) GsonHolder.getInstance().fromJson(this.payload, Class.forName(this.className));
+    }
+}
```

Comment:
Could we add a method toString here that does`GsonHolder.getInstance().toJson(serializableSystemUser)` so the clients wouldn't need to call GsonHolder?